tests/EwsContextTest.cs

```diff
@@ -13,10 +13,10 @@
 namespace EWS;
 
 [TestClass]
+[System.Diagnostics.CodeAnalysis.SuppressMessage("Reliability", "CA2000:Dispose objects before losing scope", Justification = "Code readability")]
```

Comment:
REQ: Die Rule `CA2000` kann global im Testprojekt abgeschaltet werden. Ist meiner Meinung nach in Testprojekten nicht sinnvoll.